### PR TITLE
Add Sw-Fw-Less v1.0

### PR DESCRIPTION
**Pull request by luoxiaojun1992**

### CODE REVIEW

**Comment by nbrady-techempower** [comment]
Thanks for the submission @luoxiaojun1992 

Can you please remove all of the config files that aren't related to testing? ie AMPQ, auth, etc. 

Also, can you update the `README.md` to look like: https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/Java/gemini/README.md

**Comment by luoxiaojun1992** [comment]
Thanks for your response @nbrady-techempower 
I updated the PR according to you advice.
Please review it.

File: frameworks/PHP/sw-fw-less/sw-fw-less.dockerfile [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+RUN docker-php-ext-install pdo_mysql
+
+# Bcmath extension
+RUN docker-php-ext-install bcmath
+
+# Sockets extension
+RUN docker-php-ext-install sockets
+
+# Redis extension
+RUN wget http://pecl.php.net/get/redis-${PHPREDIS_VERSION}.tgz -O /tmp/redis.tar.tgz \
```
Comment (nbrady-techempower) [inline frameworks/PHP/sw-fw-less/sw-fw-less.dockerfile]: You shouldn't be using redis for anything, right? You're not doing the cached queries test. Please remove.

File: frameworks/PHP/sw-fw-less/sw-fw-less.dockerfile [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    && ( \
+        cd hiredis \
+        && make -j$(nproc) \
+        && make install \
+        && ldconfig \
+    ) \
+    && rm -r hiredis
+
+# Swoole extension
+RUN wget https://github.com/swoole/swoole-src/archive/${SWOOLE_VERSION}.tar.gz -O swoole.tar.gz \
```
Comment (nbrady-techempower) [inline frameworks/PHP/sw-fw-less/sw-fw-less.dockerfile]: `wget -q` for logs

**Comment by luoxiaojun1992** [comment]
Modified according to your comments, please review it. @nbrady-techempower

**Pull request merged**